project/SparkBuild.scala

```diff
@@ -758,6 +774,15 @@ object Hive {
   )
 }
 
+object YARN {
+  lazy val settings = Seq(
+    excludeDependencies --= Seq(
+      ExclusionRule(organization = "com.sun.jersey"),
+      ExclusionRule("javax.servlet", "javax.servlet-api"),
+      ExclusionRule("javax.ws.rs", "jsr311-api"))
```

Comment:
Hm, so in Yarn are we adding `javax.servlet-api` back with `jakarta.servlet-api`?

Reply:
THE YARN TESTS relies on this, so I
 added it back here

Reply:
Is it needed only for tests? or during actual runtime with the regular release?

Reply:
It's for test only.

FYI, https://github.com/apache/spark/blob/317048d708465723e7008ba4ce22c2406c1aa23a/resource-managers/yarn/pom.xml#L156

Reply:
If this is for test-only, let's include this only for the test scope. It's going to introduce another problem with conflicting jars. Can we try to exclude and include this with test-only scope? If that doesn't work with SBT, that might be fine since we don't use SBT for official release process. We can add documentation for clarification.

Reply:
Do we have a JIRA ticket to track the new conflicting jars issue? I will try to fix it

Reply:
It would be able to do it at least in Maven. Nope, we don't have it. I was just post-reviewing it :-).

Reply:
Just for clarification, we're all fine as long as ...:
- `dev/deps/spark-deps-hadoop-3.2-hive-2.3` does not have the duplicated jars. This file contains the list of jars for the release.
- The tests pass

Reply:
got it

Reply:
Thank you @yaooqinn.

Reply:
my pleasure:)

Reply:
I filed a JIRA for you: SPARK-34111 :-).